ring-core/test/ring/middleware/test/file.clj

```diff
@@ -93,3 +93,46 @@
       (is (= 200 status))
       (is (= (into #{} (keys headers)) #{"Content-Length" "Last-Modified"}))
       (is (= foo-html body)))))
+
+(deftest test-wrap-file-with-prefer-handler-option
+  (let [processable-file (File. ^String public-dir "foo.bar")
+        handler (fn [request]
```

Comment:
This handler can probably be a top-level `defn` instead.

Reply:
i kept it in the let to limit its scope. it has no use outside of this test. however can move it to the top level if you want